src/_utils/item/index.unit.tsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
+import renderer from 'react-test-renderer'
 
 import Item from './index'
 import ClockIcon from 'icon/clockIcon'
 import ChevronIcon from 'icon/chevronIcon'
 
+it('Should not have changed', () => {
```

Comment:
nit: I feel "Should not have changed" is awkward and might get confused with what's expected of a feature.
=> "Should match snapshot"